test/run-conu

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+exec pytest-3 -vv -k test_s2i
```

Comment:
@TomasTomecek Please use `pytest`. If `conu` is installed from PyPi, `pytest` is also from PyPi and no `-3` suffix is there. (to note: although without -3, python3 is used)